### DISCUSSION TITLE
Second y-axis for volume?

### DISCUSSION THREAD

**Discussion by somguyth** [reactivemarkets/react-financial-charts#532] [General]
Is it possible to have a second y-axis (on the LHS of the chart) that shows how much volume is in each bar? I know we could display the volume is a separate plot, but I'm hoping to simply get a second axis displayed instead.

https://reactivemarkets.github.io/react-financial-charts/?path=/story/features-full-screen--daily

**Comment by markmcdowell** [comment]
yes you can just add another axis and set the axisAt property to left.

See https://reactivemarkets.github.io/react-financial-charts/?path=/story/features-axis--y-axis for properties

**Reply by somguyth** [reply]
I just did it successfully.

<YAxis tickFormat={this.pricesDisplayFormat} outerTickSize={0} innerTickSize={4} axisAt={"left"}/>

Thank you!
